src/libraries/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3493,8 +3493,7 @@ public static void DCS_BasicPerSerializerRoundTripAndCompare_SampleTypes_SampleI
     /// <summary>
     /// This case is a part of DCS_BasicPerSerializerRoundTripAndCompare_SampleTypes, but in these scenarios it was not support in current version.
     /// </summary>
-    [Fact]
-    [ActiveIssue("Not support")]
+    [Fact(Skip = "Not supported")]
```

Comment:
If we're going to always skip the test and there's no issue / no follow-up, shouldn't we just delete the test / dead code?

Reply:
Sure, whatever you prefer.

Reply:
Thanks. Let's delete it.

Reply:
Done